CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [CHANGE] Ingester: Do not log errors related to hitting per-instance limits to reduce resource usage when ingesters are under pressure. #5585
 * [CHANGE] gRPC clients: use default connect timeout of 5s, and therefore enable default connect backoff max delay of 5s. #5562
 * [CHANGE] The `-shutdown-delay` flag is no longer experimental. #5701
+* [FEATURE] Introduced `distributor.enable_service_overload_error_on_rate_limit` flag for configuring error code to 529 instead of 429 upon rate limit exhaustion. #5752
```

Comment:
* [FEATURE] Introduced `-distributor.service-overload-status-code-on-rate-limit-enabled` option to use status code 529 instead of 429 upon rate limit exhaustion. #5752